src/platform/graphics/webgl/webgl-graphics-device.js

```diff
@@ -797,7 +797,8 @@ class WebglGraphicsDevice extends GraphicsDevice {
     updateBackbuffer() {
 
         const resolutionChanged = this.canvas.width !== this.backBufferSize.x || this.canvas.height !== this.backBufferSize.y;
-        if (this._defaultFramebufferChanged || resolutionChanged) {
+        const isCanvasNonZero = this.canvas.width > 0 && this.canvas.height > 0;
+        if (isCanvasNonZero && (this._defaultFramebufferChanged || resolutionChanged)) {
```

Comment:
Have you tested this? I would expect this to fail as well, as the canvas is zero, our multi-sampled buffer is non-zero and we blit from one to the other.

Reply:
Yep, it doesn't throw the error. I think maybe because the canvas size can be zero, but the buffer can't

Reply:
I'm not super keen on this to be honest .. I think just reporting a warnOnce would be a better option. Otherwise the user might create a zero sized buffer and we'll keep rendering to some larger buffer. We should not hide this, just surface it.

Reply:
Right, yeah thats a valid point